### DISCUSSION TITLE
Prettification both sides completely

### DISCUSSION THREAD

**Discussion by Pierre-de-la-Verre** [WinMerge/winmerge#1622] [General]
Often I have to compare 2 HTML or JSON or XML, which have different formatting. So I have to
- select left
- mark all
- right context
- prettification
- prettification json
- select right
- mark all
- right context
- prettification
- prettification json

I'm looking for a feature / button / ... to make this shorter: "Prettify both files for JSON" - 1or 2 commands instead of 10.

Does it already exist? Can it be added?

**Comment by sdottaka** [comment]
If you are comparing two JSON files, try using the PrettifyJSON unpacker plugin as below.

If you are pasting JSON data from the clipboard and comparing it, unfortunately it is currently not possible to apply the plugin to both the left and right panes at the same time.

![image](https://user-images.githubusercontent.com/98126/208708627-83a87137-8ade-44f7-8484-6db118b12fcf.png)

**Reply by Pierre-de-la-Verre** [reply]
Thanks, the command does what I want.
A button would do the missing 5% of functionality.
